CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 All changes to the project will be documented in this file.
 
-## [1.34.9] - not yet released
+## [1.34.9] - 2019-12-10
```

Comment:
Oh would you mind adding to the Razor section that we've updated colorization of .razor and .cshtml files? I forgot to do it in my PR 😁

Reply:
this would go in the omnisharp-vscode repo though

Reply:
yup, that's what I get for mixing up my notifications 😆